sky/shell/shell.cc

```diff
@@ -16,7 +16,9 @@
 #include "mojo/message_pump/message_pump_mojo.h"
 #include "sky/shell/ui/engine.h"
 #include "sky/shell/switches.h"
+#if !defined(OS_LINUX)
 #include "ui/gl/gl_surface.h"
```

Comment:
I might as well, because I'm pretty sure I will have to gn gen for all platforms as a sanity check that I have the gn deps stuff right.